helpers/interfaces.ts

```diff
@@ -0,0 +1,4 @@
+export interface User{
```

Comment:
Add `default` to use it same as other classes imports, without {}

Reply:
done:)